src/pages/Components/RatingStars/code.md

```diff
@@ -0,0 +1,179 @@
+---
+title: 'Code'
+order: 2
+---
+
+> Rating stars can be used in two ways : (1.) when filing or amending an opinion (2.) when displaying the overall score on a product sheet.
+
+Implementing rating stars :
+
+- [When filing or amending an opinion](#case-1)
+- [When displaying the overall score](#case-2)
+
+<a name="case-1"></a>
```

Comment:
?

Reply:
didn't understand #case-1 and #case-2 what's the difference?

Reply:
These are anchors to be positioned in the right place in the page.
#case-1 to go to the "When filing or amending an opinion" section
#case-2 to go to the "When displaying the overall score" section.

However, since Gael asked to delete the table of contents, I will also delete these parts.